man/man1/fzf.1

```diff
@@ -1547,6 +1651,9 @@ e.g.
 .br
 or any single character
 
+Note that some terminal emulators (such as \fIkitty\fR, \fIiterm2\fR, etc) do not distinguish between
+CTRL-* and those without CTRL and therefore these bindings may not work.
```

Comment:
I use Kitty on macOS. Which terminal emulator should I use to test these?

Reply:
I use gnome-terminal on linux. But I successfully tested this with the following terminals:

- ghostty
Use `keybind = clear` in `~/.config/ghostty/config` (to prevent default shortcuts intercepting events)
- wezterm
Use `config.disable_default_key_bindings = true` in `~/.wezterm.lua`
- kitty
Use `clear_all_shortcuts yes` in `~/.config/kitty/kitty.conf`

You might need to specify an option to use the mac option key as alt. Like `macos-option-as-alt = left` in ghostty.

Reply:
> "Note that some terminal emulators (such as \fIkitty\fR, \fIiterm2\fR, etc) do not distinguish between
CTRL-* and those without CTRL and therefore these bindings may not work."

I don't think that's true, at least not for Kitty. And even though I can't test it, it's hard to believe for iterm2.